tests/Integration/OtherTest.php

```diff
@@ -226,7 +226,7 @@ public function test_corrupted_options(): void {
 		update_option( Parsely::OPTIONS_KEY, 'someinvalidvalue' );
 
 		$options = self::$parsely->get_options();
-		self::assertSame( self::EMPTY_DEFAULT_OPTIONS, $options );
```

Comment:
We don't need the duplicated default settings in tests because we can use the one from `Parsely`